capstone/src/main/webapp/restaurantSingUp.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>STEP Capstone</title>
+    <script src="signUp.js"></script>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+  </head>
+  <body>
+    <h1>Sign Up as a Restaurant!</h1>
+    <div id="sign-up">
+      <p>Name:</p>
+      <textarea id="name"></textarea>
+
+      <p>Cuisine:</p>
+      <textarea id="cuisine"></textarea>
```

Comment:
Are we going to just direct users to enter cuisines as a comma-separated list for now?

Reply:
That's a good question. I was hoping for it to be a tiered choice system at some point, so that I could use whether a certain "kind" of food is primary or secondary as part of the sorting for the search index.

Reply:
So for now in the backend I set it up to handle csv but we can changed that. I feel we should talk about that in a meeting.